data/tlg0007/tlg073/tlg0007.tlg073.perseus-eng4.xml

```diff
@@ -0,0 +1,480 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+	<teiHeader xml:lang="eng">
+		<fileDesc>
+			<titleStmt>
+				<title type="work">Of Large Acquaintance: or, an Essay to Prove the Folly of Seeking Many Friends</title>
```

Comment:
we don't use a type=work here (we could but we haven't been doing this consistently)